### DISCUSSION TITLE
关于写入二进制数据到mysql中

### DISCUSSION THREAD

**Discussion by lsx1994** [cloudwu/skynet#1647] [Q&A] [answered]
我现在做法是先 `serialize.serialize_string`转字符串， 然后用 mysql里的`strgsub(str, "[\0\b\n\r\t\26\\\'\"]", escape_map)`处理特殊字符后写入mysql，但是这样消耗的时间翻倍了，一般是怎么处理的？

如果不处理特殊字符，写入mysql的时候string.format会报错。

**Comment by sniper00** [comment, marked as answer]
base64

**Reply by lsx1994** [reply]
感谢，试了下base64 就加了几毫秒。
